caf-audit/src/test/java/com/hpe/caf/auditing/elastic/GeneratedAuditLogIT.java

```diff
@@ -38,9 +39,12 @@
 import java.util.Date;
 import java.util.Map;
 import java.util.UUID;
+import java.util.concurrent.ExecutionException;
 
 public class GeneratedAuditLogIT {
 
+    private static final String testTenant = "tenant1";
+
     private static final Logger LOG = LoggerFactory.getLogger(GeneratedAuditLogIT.class);

```

Comment:
LOG is never used.  I suggest we use it.
If not used it should be removed, but I'd rather see it used.  @CJComac use it where appropriate in your tests, you don't need to retro-fit it.